components/FeaturedStoryPreview.js

```diff
@@ -0,0 +1,28 @@
+import PageLink from '../components/PageLink';
+import { H3, P } from '../style/typography';
+import { getString } from '../lib/richText';
+
+export default function FeaturedStoryPreview({ featuredStoryData }) {
+  const {
+    uid: uid,
+    preview_title_regular: previewTitleRegular,
+    preview_title_italic: previewTitleItalic,
+    preview_text: previewText,
+    preview_image: previewImage,
+    signature,
+  } = featuredStoryData;
+  return (
+    <>
+      <H3>{getString(previewTitleRegular)}</H3>
+      <H3>{getString(previewTitleItalic)}</H3>
+      <P>{getString(previewText)}</P>
+      <img
+        style={{ width: '500px' }}
```

Comment:
Can you pull out this style please?

Reply:
Decided to remove the styling since that will all be done in a separate PR, good catch thank you